### DISCUSSION TITLE
ReSpeaker Mic Array v2.0 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1521] [Q&A]
# ReSpeaker Mic Array v2.0 | Seeed Studio Wiki

ReSpeaker Mic Array v2.0

https://wiki.seeedstudio.com/ReSpeaker_Mic_Array_v2.0/

**Comment by ValdiW** [comment]
Hi, with ReSpeacer Mic Array 2.0 is there any way to extract microphone(s) signal ( processed or unprocessed, sum of 4 channels ) and play it through audio output ( jack connector ) without using a computer ( while USB is beeing used only to provide power ) ?
Thanks

**Comment by LeeHieu134** [comment]
Hi All,
After flashing with:

sudo python dfu.py --download MicArrayV3_firmware/6_channels_dfu_4.0.0_firmware.bin

my device no longer enumerates in DFU mode. How can I recover/force it back into DFU? I tried:
sudo python dfu.py --download 6_channels_firmware.bin
and it didn't work. I'm using v2.0 and just want it to work like it did initially.
please help me!

**Reply by nfs0619** [reply]
Hi,
We recommend that you contact our technical support team at **[techsupport@seeed.io](mailto:techsupport@seeed.io)**. They will be able to provide you with faster and more detailed guidance.
